### PR TITLE
Updated to reflect latest rfc meeting

### PR DESCRIPTION
**Pull request by doddi**

### CODE REVIEW

File: accepted/0027-audit-metadata.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+Developers and users who are using the npm audit tool, may want to get additional or finely tuned report information about their project.
+For developers who perform `npm audit` or `npm install` with third-party applications there is no way to provide additional metadata.
+Like `npm-session` is used by the npm registry to identify individual user sessions, it would be useful to provide a generic way for a maintainer to add additional metadata to their project  which is sent during an `npm audit` or `npm install`.
+
+## Rationale and Alternatives
+
+* **Rationale:** 
+  * This is easy to implement solution to provide such additional value in the HTTP request.
+  * Building this functionality will allow developers to easily set additional information that can be passed along in a heaer for third-party application use
```
Comment (Mykyta) [inline accepted/0027-audit-metadata.md]: typo in the "heaer"

File: accepted/0027-audit-metadata.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  
+* **Alternatives:** 
+  * Use `preshrinkwrap` or/and `postshrinkwrap` scripts to install a custom value to the project like the `app-id`
+
+## Implementation
+
+* Define a new `metadata` object that can be added to the package.json file `metadata`.
```
Comment (Mykyta) [inline accepted/0027-audit-metadata.md]: "metadata" at the end of the sentence is redundant?

**Review by Mykyta — COMMENTED** [review]
Looks good

File: accepted/0027-audit-metadata.md (hunk 2 above)
Comment (doddi) [reply accepted/0027-audit-metadata.md]: https://github.com/Mykyta/rfcs/pull/1/commits/887ef308e3d14ef401a0f2cc041bfccff9ac30df

File: accepted/0027-audit-metadata.md (hunk 1 above)
Comment (doddi) [reply accepted/0027-audit-metadata.md]: https://github.com/Mykyta/rfcs/pull/1/commits/887ef308e3d14ef401a0f2cc041bfccff9ac30df

**Review by Mykyta — APPROVED** [review]

**Pull request merged**